app/src/main/res/xml/grayscale_icon_map.xml

```diff
@@ -52,6 +52,7 @@
     <icon drawable="@drawable/audiofx" package="org.lineageos.audiofx" name="AudioFX" />
     <icon drawable="@drawable/aurora_droid" package="com.aurora.adroid" name="Aurora Droid" />
     <icon drawable="@drawable/aurora_store" package="com.aurora.store" name="Aurora Store" />
+    <icon drawable="@drawable/ausweisapp2" package="com.governikus.ausweisapp2" name="AusweisApp2" />
```

Comment:
<icon drawable="@drawable/ausweisapp2" package="com.governikus.ausweisapp2.dev" name="AusweisApp2" />

This is the beta version. Maybe you can add it

Reply:
Wdym exactly? `com.governikus.ausweisapp2` is a beta version?

Reply:
com.governikus.ausweisapp2.dev is the beta package name if you wanna add it too

Reply:
sure give me 1 second

Reply:
Sry forgot `` 
`<icon drawable="@drawable/ausweisapp2" package="com.governikus.ausweisapp2.dev" name="AusweisApp2" />`